lib/routes/account.js

```diff
@@ -223,8 +225,16 @@ module.exports = (
                 return log.notifyAttachedServices('verified', request, {
                   email: account.email,
                   uid: account.uid,
-                  locale: account.locale
+                  locale: account.locale,
+                  marketingOptIn: marketingOptIn,
                 })
+              } else if (marketingOptIn) {
+                return marketingCache.set(account.uid, true)
+                  .catch(err => log.error({
```

Comment:
Probably, huh. I can't think of a reason we'd want the error to fail the whole thing...